modules/localstorage-onload.js

```diff
@@ -0,0 +1,14 @@
+import library from './library.js';
+import drawOnScreen from './drow-on-screen.js';
+
+export default function localStorageOnLoad() {
+  window.addEventListener('load', () => {
+    const localStorageItem = window.localStorage.getItem('books');
+    if (localStorageItem) {
+      library.booksArr = JSON.parse(localStorageItem);
+      library.booksArr.forEach((element) => {
+        drawOnScreen(element);
+      });
+    }
+  });
+}
```

Comment:
- [ ] Good job working on this function 👍. As I suggested above please change it to an arrow function syntax.